scripts/start_docker_bblfsh.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# CI convenience script to start a bblfsh server in Docker container.
+# Skips starting server for jobs that do not need it i.e stylecheck
+
+E_BAD_DOCKER=2
+image="bblfsh/bblfshd:v2.4.2"
+
+set -ev
+
+if [[ -z "${STYLE_CHECK}" ]]; then
+  echo "Starting Docker image ${image}"
+  docker run --name bblfshd --privileged -p 9432:9432 -d "${image}"
+
+  if [[ "$?" -ne 0 ]]; then
+    echo "Unable to start Docker ${file_list}" >&2
+    exit "${E_BAD_DOCKER}"
+  fi
+
+  docker exec -it bblfshd bblfshctl driver install --recommended
+else
+  echo "Skip starting Docker image ${image}"
+fi
+
+docker ps
```

Comment:
I understand this is not strictly related to this PR, since you were already doing this previously with `scripts/start_docker_db.sh`, but you might you docker-compose for this.

It can be easily pre-installed in Travis ([see here](https://docs.travis-ci.com/user/docker/#Using-Docker-Compose) and it's a single file with more concise syntax to define Docker dependencies. You can choose to start all of them with `up` or specific `up bblfsh`.

cc @bzz

Reply:
yes. We are going to have 3 docker images (db, bblfsh, feature extraction) in dependencies for gemini. It makes perfect sense to use docker-compose in the feature. (actually, I already do locally)
Make it should be part of new release preparation issue?